<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下拉式菜单示例</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>下拉式菜单</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>字体</string>
    </property>
    <widget class="QMenu" name="menu_text">
     <property name="title">
      <string>文本风格</string>
     </property>
     <addaction name="action_1"/>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
    </widget>
    <addaction name="action_name"/>
    <addaction name="menu_text"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>颜色</string>
    </property>
    <addaction name="actioncolor"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>结束</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_name">
   <property name="text">
    <string>字体名称</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="text">
    <string>颜色设置</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
